db/view/base_info.hh

```diff
@@ -42,21 +30,11 @@ public:
     // succeeding to reliably build the former.
     bool has_base_non_pk_columns_in_view_pk;
 
-    // If base_non_pk_columns_in_view_pk couldn't reliably be built, this base
-    // info can't be used for computing view updates, only for reading the materialized
-    // view.
-    bool use_only_for_reads;
```

Comment:
I really hope this use_only_for_reads is indeed no longer needed. It always looked like a kludge.